fabiopellosowj/magento2
# Fix gallery disabled attribute

### Description

Removed 'disabled' field from catalog_product_entity_media_gallery table because is not used.

### Fixed Issues (if relevant)
1. magento/magento2#17796: Seems that `catalog_product_entity_media_gallery` contains unused `disabled` field

### Manual testing scenarios

1. Create New Product;
2. Add an Image to your product; 
3. Save the product;
4. Check if the table catalog_product_entity_media_gallery not have the field 'disabled';
